util/mock/context.go

```diff
@@ -174,11 +179,6 @@ func (c *Context) Cancel() {
 	c.cancel()
 }
 
-// GoCtx returns standard context.Context that bind with current transaction.
```

Comment:
Why remove this function?

Reply:
It's never used...
This function is a dummy implementation of `context.Context` interface, but the interface changed and removed this method.